os/include/arpa/inet.h

```diff
@@ -199,6 +211,9 @@ uint16_t htons(uint16_t hs);
 /**
  * @brief Convert the Internet host address from the IPv4 numbers-and-dots notation into binary form
  *
+ * @details @b #include <arpa/inet.h>\n
+ * SYSTEM CALL API\n
```

Comment:
please check if this API is SYSTEM CALL or not.

Reply:
revised